libs/hdf-converters/index.ts

```diff
@@ -1,6 +1,7 @@
 export {ASFFResults} from './src/asff-mapper/asff-mapper';
 export * from './src/aws-config-mapper';
 export * from './src/burpsuite-mapper';
+export * from './src/checklist-intermediary-mapper';
```

Comment:
This is all fine for now since it'll be replaced by the checklist mapper pr.  Intent is for this intermediary format to still be available but in the passthrough of an hdf instead of directly.

Reply:
Replaced after merge from main with checklist mapper